docs/content/scripting/examples/callback.qml

```diff
@@ -0,0 +1,39 @@
+import QtQml 2.0
+import com.qownnotes.noteapi 1.0
+
+/**
+ * This script is just an example of how to use scripts
+ * Visit http://docs.qownnotes.org/ for more information about scripting
+ */
+QtObject {
```

Comment:
Please use `Script` (like in the newer scripts)

Reply:
I'm getting this for `Script`:
```
Script is not a type Script { ^)
```

Reply:
Is this missing?

```
import QtQml 2.0
import QOwnNotesTypes 1.0
```

Reply:
Yes, `QOwnNotesTypes 1.0` was missing. It's missing in `example-script` I used as base.
What is the advantage of Script compared to the QtObject parser?

Reply:
Just to make sure everyone understands that the QML is a script (for QOwnNotes).